assets/style.css

```diff
@@ -3,9 +3,23 @@ UI Styling
 */
 
 .background-image {
-  width: 100%;
-  height: 100%;
-  position: fixed;
-  z-index: -1;
-  opacity: 0.4;
-}
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: -1;
```

Comment:
Rather than using a negative z-index, we should be able to add the background image to that .container div through css

Reply:
Oh you're saying to add that background image through css
We can do that

Reply:
Hm so this might not work as expected. If we attach it to the container it actually only fills the background of the container as opposed to the whole page. So the alternative would be to attach it to the body, which fills the whole page, but then we can't give it an opacity and correct sizing without adjusting everything in the body to the same properties and we can't specifically target it since it is a css property, not an HTML element

<img width="1162" alt="Screen Shot 2020-08-20 at 11 38 06 AM" src="https://user-images.githubusercontent.com/63083817/90811530-a0191380-e2d9-11ea-9924-59c59d73d6e3.png">

<img width="1160" alt="Screen Shot 2020-08-20 at 11 35 50 AM" src="https://user-images.githubusercontent.com/63083817/90811347-4f091f80-e2d9-11ea-9fa6-f9d056d650d1.png">